### PR TITLE
中国語のREADME_CN.mdを追加。

### PR DESCRIPTION
**Pull request by xia-mc**
## 内容
- README.mdを中国語に翻訳。

## 変更点
- README_CN.mdファイルを作成する。

## チェックリスト:
-   [x] このリポジトリのコードスタイルに沿っているか
-   [x] 自分自身で動作確認を行ったか、また、それは正常に動作したか

### CODE REVIEW

**Comment by ringo360** [comment]
Thanks for your contribution!
I'll check now👀

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+**例:**
+```js
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('hello')
+        .setDescription('Hello World!'),
+    execute: async function (interaction) {
+        await interaction.reply('Hello World!') //処理を記述
```
Comment (ringo360) [inline README_CN.md]: If possible, I would appreciate a translation here as well 📝

File: README_CN.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+const upload = require('./upload');
+
+class ExampleFeature {
+	onLoad() {
+		CommandManager.default.addCommands({
+            data: new SlashCommandBuilder()
+                .setName('hello')
+                .setDescription('Hello World!'),
+            execute: async function (interaction) {
+                await interaction.reply('Hello World!') //処理を記述
```
Comment (ringo360) [inline README_CN.md]: same as before

**Review by ringo360 — CHANGES_REQUESTED** [review]
I checked your PR!
Thanks! but there are several areas that we would like to see corrected.

**Comment by xia-mc** [comment]
> I checked your PR! Thanks! but there are several areas that we would like to see corrected.

Thanks! I'll fix it now.

**Review by ringo360 — APPROVED** [review]
Okay!

**Comment by ringo360** [comment]
I'll merge this PR! 🚀
Thank you for your contribution ❤️

**Pull request merged**